tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -120,26 +120,29 @@ private[scalanative] object LLVM {
       }
     }
 
+    val (includePaths, excludePaths) = paths.partition(include(_))
+
     // delete .o files for all excluded source files
-    paths.foreach { path =>
-      if (!include(path)) {
-        val opath = Paths.get(path + oExt)
-        if (Files.exists(opath)) {
-          Files.delete(opath)
-        }
-      }
+    excludePaths.foreach { path =>
+      val opath = Paths.get(path + oExt)
+      if (Files.exists(opath)) Files.delete(opath)
```

Comment:
Side-effecting branches of if's should be on their own line, so that they stand out:
```suggestion
      if (Files.exists(opath))
        Files.delete(opath)
```

Reply:
Done, good to know.